scripts/smoke_tests/commands_test.ps1

```diff
@@ -79,27 +129,104 @@ $commands += "az iot dps enrollment show -g $resource_group_name --dps-name $dps
+$commands += "az dt endpoint wait --created -n $dt_instance_name -g $resource_group_name --en $dt_eventgrid_endpoint --interval 1"
+
+$commands += "az dt route create -n $dt_instance_name -g $resource_group_name --endpoint-name $dt_eventgrid_endpoint --route-name $dt_route_name"
+$commands += "az dt route show -n $dt_instance_name --route-name $dt_route_name"
+
+$commands += "az dt model create -n $dt_instance_name --models $dtmi_model_content"
+$commands += "az dt model show -n $dt_instance_name --dtmi 'dtmi:com:example:Floor;1' --definition"
+
+$commands += "az dt twin create -n $dt_instance_name --dtmi 'dtmi:com:example:Floor;1' --twin-id $dt_twin_id"
+$commands += "az dt twin show -n $dt_instance_name --twin-id $dt_twin_id"
+$commands += "az dt twin query -n $dt_instance_name -q 'select * from digitaltwins' --show-cost"
+$commands += "az dt twin telemetry send -n $dt_instance_name --twin-id $dt_twin_id"
+$commands += "az dt model create -n $dt_instance_name --models $dtmi_target_model_content"
+$commands += "az dt twin create -n $dt_instance_name --dtmi 'dtmi:com:example:Room;1' --twin-id $dt_target_twin_id"
+$commands += "az dt twin relationship create -n $dt_instance_name --relationship-id $dt_twin_relationship_id --relationship contains --twin-id $dt_twin_id --target $dt_target_twin_id"
+$commands += "az dt twin relationship show -n $dt_instance_name --twin-id $dt_twin_id --relationship-id $dt_twin_relationship_id"
+
+# Digital Twins Resource Cleanup
+$commands += "az dt model delete -n $dt_instance_name --dtmi 'dtmi:com:example:Floor;1'"
+$commands += "az dt model delete -n $dt_instance_name --dtmi 'dtmi:com:example:Room;1'"
+$commands += "az dt twin relationship delete-all -n $dt_instance_name --twin-id $dt_twin_id -y"
+$commands += "az dt twin delete -n $dt_instance_name --twin-id $dt_twin_id"
+$commands += "az dt twin delete -n $dt_instance_name --twin-id $dt_target_twin_id"
+$commands += "az dt route delete -n $dt_instance_name --route-name $dt_route_name"
+$commands += "az dt endpoint delete -n $dt_instance_name -g $resource_group_name --en $dt_eventgrid_endpoint -y"
+$commands += "az eventgrid topic delete --name $dt_eventgrid_topic --resource-group $resource_group_name"
+$commands += "az dt reset -n $dt_instance_name -y"
+$commands += "az dt delete -n $dt_instance_name -y"
+
+# IoT Central
+$commands += "az iot central device-template create --app-id $central_app_id --device-template-id $central_device_template_id --content $central_device_template_content"
+$commands += "az iot central device-template show --app-id $central_app_id --device-template-id $central_device_template_id"
+
+$commands += "az iot central device create --app-id $central_app_id --device-id $central_device_id --template $central_device_template_id --simulated"
+# Sleeping to ensure device creation is completed
+$commands += "Start-Sleep -s 30"
```

Comment:
Is this sleep required for further device oriented commands to work?

Reply:
Correct, we need the sleep operation to ensure that device is initialized so that central device commands work.